data/src/test/java/org/apache/iceberg/io/TestWriterMetrics.java

```diff
@@ -128,25 +130,25 @@ public void verifySortedColMetric() throws Exception {
 
     // Only two sorted fields (id, structField.longValue) will have metrics
     Map<Integer, ByteBuffer> lowerBounds = dataFile.lowerBounds();
-    Assert.assertEquals(
-        3, (int) Conversions.fromByteBuffer(Types.IntegerType.get(), lowerBounds.get(1)));
-    Assert.assertFalse(lowerBounds.containsKey(2));
-    Assert.assertFalse(lowerBounds.containsKey(3));
-    Assert.assertFalse(lowerBounds.containsKey(4));
-    Assert.assertEquals(
-        3L, (long) Conversions.fromByteBuffer(Types.LongType.get(), lowerBounds.get(5)));
+    assertThat((int) Conversions.fromByteBuffer(Types.IntegerType.get(), lowerBounds.get(1)))
+        .isEqualTo(3);
+    assertThat(lowerBounds).doesNotContainKey(2);
```

Comment:
nit: you can chain them all together to have `assertThat(..).doesNotContainKey(2).doesNotContainKey(3)....`

Reply:
Cool, but keeping them on separate lines makes it actually more readable IMHO. Pls LMK if I should change this.

Reply:
I'm ok either way and it doesn't make a big difference one way or another, so I'm letting you decide how you want to have them